task1/map.c

```diff
@@ -15,22 +15,26 @@ enum cardd map[9];
 // The function set_dir should put the value dir at position x, y into the array map
 // Check x, y and dir for validity
 
-// i would like to note that i switched x and y here because i cant let x go downwards and y to the left, it just feels wrong
+// I would like to note that i switched x and y here because i cant let x go downwards and y to the left, it just feels wrong
```

Comment:
Does not the y component goes from left to right. Therefore y goes to the right.

Reply:
Technically yeah, but i'll let it stay like this for now as it's further explained a few more lines further down